README.md

```diff
@@ -11,7 +11,8 @@ For more information, [visit our website](https://musicbrainz.org/doc/About).
 To get help, please join #metabrainz on irc.freenode.net
 
 Important announcements (e.g. about breaking changes to the schema or our 
-API / web service) will be made on [our blog](https://blog.metabrainz.org/),
+API / web service) will be made on
+[our blog](https://blog.metabrainz.org/category/development/updates/breaking-changes/),
```

Comment:
Link more specifically to MusicBrainz breaking changes:

```suggestion
[our blog](https://blog.metabrainz.org/category/musicbrainz+breaking-changes/),
```

I guess that [2020-09-21](https://blog.metabrainz.org/2020/09/21/musicbrainz-server-update-2020-09-21/) should be tag with breaking changes then?

Reply:
I don't think that link works as intended? I'm seeing a lot of stuff not tagged with breaking changes.

Reply:
It works as intended: it shows all previous posts about musicbrainz schema changes as I moved “Schema Change Release” category under “Breaking Changes”.

Reply:
Ooooooh. Magic. Ok then!